src/pipe/gui/GuiFrame.java

```diff
@@ -1220,6 +1289,12 @@ public void registerViewActions(@NotNull List<GuiAction> viewActions) {
             showTokenAgeCheckBox.setVisible(true);
             showDelayEnabledTransitionsCheckbox.setVisible(true);
         }
+

```

Comment:
Now that we add more logic here, the refactor of the method should be completed and logic moved to tabContent